src/groundwater/aquifer.jl

```diff
@@ -149,6 +152,14 @@ function harmonicmean_conductance(k1, k2, H1, H2, l1, l2, width)
     end
 end
 
+function harmonicmean_conductance(k1, k2, l1, l2, width)
```

Comment:
Suggest to work with one `harmonicmean_conductance` function, it is now a bit confusing with two very similar functions. Probably best to also use this version for the other case (uniform k value).